includes/admin/class-wc-rest-payments-webhook-controller.php

```diff
@@ -107,6 +107,8 @@ public function handle_webhook( $request ) {
 				. var_export( WC_Payments_Utils::redact_array( $body, WC_Payments_API_Client::API_KEYS_TO_REDACT ), true ) // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_var_export
 			);
 
+			do_action( 'woocommerce_payments_before_webhook_delivery', $event_type, $body );
```

Comment:
We should probably wrap this call into an individual try-catch statement (with logging), so the main workflow is not getting interrupted.

Reply:
Done with https://github.com/Automattic/woocommerce-payments/pull/1144/commits/c0703b82579c3ae81c96f7640d841295fe193642

Reply:
Awesome, thank you